api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1875,7 +1874,7 @@ public void makeBucket(String bucketName, String region)
       configString = config.toString();
     }
 
-    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, 0);
+    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, configString.length());
```

Comment:
This is for creating bucket, do we really need to convert to UTF8 ?

As @balamurugana, suggested, we need to enhance createRequest for a simpler logic.